concourse-chrome-driver/terraform/deployments/670214072732/variables.tf

```diff
@@ -0,0 +1,84 @@
+variable "codestar_connection_id" {
+  type    = string
+  default = "51c5be90-8c8f-4d32-8be4-18b8f05c802c"
+}
+
+variable "codebuild_service_role_name" {
+  description = "the role code build uses to access other AWS services"
+  type        = string
+  default     = "CodePipelineExecutionRole"
+}
+
+variable "docker_hub_repo" {
+  description = "Docker Hub repository"
+  type        = string
+  default     = ""
+}
+
+variable "docker_hub_credentials" {
+  description = "Name of the secret in SSM that stores the Docker Hub credentials"
+  type        = string
+  default     = "docker_hub_credentials"
+}
+
+variable "stage_name" {
```

Comment:
I don't think these should be required. These refer to the stage name and action name in the codepipeline to be passed to the individual codebuild actions.

Reply:
Ta removed.